test/images/scripts/debian-testing.setup

```diff
@@ -48,14 +47,15 @@ virtinst \
 useradd -m -U -c Administrator -G sudo -s /bin/bash admin
 echo admin:foobar | chpasswd
 
-# For storaged
+# For latest udisks
 echo "deb http://deb.debian.org/debian experimental main" >/etc/apt/sources.list.d/experimental.list
 # FIXME: we need to enable jessie-backports until docker.io gets back into testing
 echo "deb http://deb.debian.org/debian jessie-backports main" >/etc/apt/sources.list.d/jessie-backports.list
 apt-get -y update
 
 export DEBIAN_FRONTEND=noninteractive
 eatmydata apt-get -y install $TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES
+eatmydata apt-get -y install -t experimental udisks2
```

Comment:
Ahh, I see why it is necessary.  Would it make sense to introduce a COCKPIT_EXPERIMENTAL_DEPS variable instead of hard coding udisks2 here?  That would make things crystal clear.